sway-core/src/ir_generation/convert.rs

```diff
@@ -145,6 +145,9 @@ fn convert_resolved_type(
         }
         TypeInfo::RawUntypedPtr => Type::get_uint64(context),
         TypeInfo::RawUntypedSlice => Type::get_slice(context),
+        // TODO: Are these ok?
```

Comment:
You'll want to remove this or clarify what's left to do here.

Reply:
Removed.